pybossa/cache/site_stats.py

```diff
@@ -421,3 +421,53 @@ def management_dashboard_stats_cached():
         # reset stats for any missing stat
         map(delete_cache_group, management_dashboard_stats)
     return stats_cached
+
+def total_tasks(interval):
+    """
+    Total tasks
+    """
+    sql = text(f'''
+        SELECT count(task.id) as num_tasks
+        FROM task WHERE
+        to_timestamp(task.created, 'YYYY-MM-DD"T"HH24:MI:SS.US') > current_timestamp - interval '{interval}';
+    ''')
+    rows = session.execute(sql).fetchall()
+    return rows[0][0]
+
+def total_taskruns(interval):
```

Comment:
Similarly, we have ```n_total_tasks_site```.

Reply:
This method sums the total number of answers on each task and returns the result. Is this guaranteed to equal the number of task runs?

Reply:
No, it's not. The number of task runs is the number of completed answers. sums of the total number(potential total # of task runs) is greater than # total task runs. We probably only need one of them, not both.

Reply:
so if the values are not the same, is it ok to use `n_total_tasks_site` rather than creating a new method to count the number of taskruns?

Reply:
Let's use ```n_task_runs_site```